### ISSUE TITLE
Minified source css gets beautified

### ISSUE THREAD

**Issue by SimonVerboven** [frontendfriends/grunt-combine-mq#7]
I noticed that my minified css file that I supplied as a source gets beautified after the media queries get combined through this task.

I know that in issue #4 you mentioned you added support for beautification, but I would expect this to be an option, rather than a feature behind the screens.

Maybe I'm missing something and it can indeed be configured, but I can't it in the tasks' documentation at least. 

In short: I want to be able to keep my minified css minified after running this task. Is this possible? If not, could you please make it possible?
